doc/en/components/inputs/input.md

```diff
@@ -0,0 +1,189 @@
+_keywords: $Platform$ form, $ProductName$, Infragistics
+mentionedTypes: ['Form']
+---
+# $Platform$ Input Overview
+
+The $ProductName$ Input is a component where the user can enter data.
+
+## $Platform$ Input Example
+
+<div class="divider--half"></div>
+
+<code-view style="height: 100px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/input-overview"
+           alt="$Platform$ Input Example"
+           github-src="inputs/input/overview">
+</code-view>
+
+## Dependencies
+
+<!-- Blazor -->
+
+To get started with the Input component, you first need to register its module.
+
+```razor
+IgbInputModule.Register(IgniteUIBlazor);
+```
+
+<!-- end: Blazor -->
+
+<div class="divider--half"></div>
+
+<!-- WebComponents -->
+
+To get started we need to import the `Input` in our typescript file and register the component by calling the [`defineComponents()`]({environment:wcApiUrl}/index.html#defineComponents) function as follows:
+
+```ts
+import { defineComponents, IgcInputComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcInputComponent);
+```
+
+<!-- end: WebComponents -->
+
+After we import the `Input` component we are ready to start using it, so let's add our first Input.
+
+```html
+<igc-input type="email" label="Subscribe" placeholder="john.doe@mail.com"></igc-input>
+```
+
+## Prefix & Suffix
+
+With `prefix` and `suffix` slots we can add different content before and after the main content of the Input. In the following sample we will create a new Input field with a text prefix and an icon suffix:
+
+```html
+<igc-input type="tel" label="Phone" placeholder="888 123456">
+  <span slot="prefix">+359</span>
+  <igc-icon name="phone" slot="suffix"></igc-icon>
+</igc-input>
+```
+
+<div class="sample-container loading" style="height: 80px; width: 400px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/input-prefix-suffix">
+</iframe></div>
+
+## Helper Text
+
+The `helper-text` slot provides a hint placed below the Input. Let's add a helper text to our phone Input:
+
+```html
+<igc-input type="tel" label="Phone">
+  <span slot="prefix">+359</span>
+  <igc-icon name="phone" slot="suffix"></igc-icon>
+  <span slot="helper-text">Ex.: +359 888 123 456</span>
+</igc-input>
+```
+
+<div class="sample-container loading" style="height: 100px; width: 400px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/input-helper-text">
+</iframe></div>
+
+## Input Sizing
+
+We can allow the user to choose the size of the `Input` by using its `Size` property. То do this, we will add some radio buttons to display all size values. This way whenever one gets selected, we will change the size of the Input.
+
+```ts
```

Comment:
As per Martin's guidance, we should avoid adding code snippets like these. We'd rather add a code-view with an actual sample instead, because this way we can have proper code for the different platforms. In this case, let's just explain what exactly they are seing in the demo below.